utils/build.ps1

```diff
@@ -2799,9 +2799,10 @@ if (-not $SkipBuild) {
     Invoke-BuildStep Build-XML2 Windows $Arch
     Invoke-BuildStep Build-CURL Windows $Arch
     Invoke-BuildStep Build-LLVM Windows $Arch
-
-    # Build platform: SDK, Redist and XCTest
     Invoke-BuildStep Build-Runtime Windows $Arch
+  }
+  foreach ($Arch in $WindowsSDKArchs) {
+    # Build platform: SDK, Redist and XCTest
```

Comment:
Can you explain why this is needed? We always iterate the SDK arches, so that architecture's runtime is always built before the rest of the SDK. I'm wondering if we need to split this up or if there is something else that is hiding here.

Reply:
No, that doesn't make sense. If the compiler requires the runtime, how did it link against the runtime before it was available? It depends on the _old_ (pinned) runtime, not the new runtime. This is a path issue.

Reply:
Done